cluster/gce/config-common.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 # Copyright 2016 The Kubernetes Authors.
+# Copyright 2020 Authors of Arktos - file modified.
```

Comment:
this is likely a glitch in the tool to update the lic headers. there is no changes in the file in the PR. so please remove this file.

Reply:
just removed this file.